HISTORY.rst

```diff
@@ -5,6 +5,8 @@ History
 Unreleased
 ----------
 
+* Add support for generating Javascript code.
```

Comment:
Should we call out that this is specifically for glean.js (to avoid confusion with Javascript-in-FOG?)